uint32 waypoint_id
---
bool success
string message 